src/bundle/Resources/public/scss/_view.content.scss

```diff
@@ -0,0 +1,14 @@
+.ez-content-view {
```

Comment:
The files should be named `_content-view.scss`. Here the pattern is with `-`.

Reply:
I disagree. In this repo we have a convention with dots.